arlo-client/src/components/MultiJurisdictionAudit/RoundManagement/index.tsx

```diff
@@ -31,17 +31,17 @@ const RoundManagement = ({ round, auditBoards, createAuditBoards }: IProps) => {
     jurisdictionId: string
   }>()
 
-  const [ballots, setBallots] = useState<IBallot[]>([])
+  const [ballots, setBallots] = useState<IBallot[] | null>(null)
```

Comment:
is this `null` alternative necessary? Are we differentiating between an empty ballot list and `null`? Do we need to?

Reply:
`null` means we're loading the data. it's a pattern we've started to establish all across the frontend. empty lists might have diff meanings in diff places, so i think it's helpful to be consistent. (e.g. does an empty ballot list mean this jurisdiction wasn't assigned any ballots? or does it mean we're loading the data still?)